lib/packwerk/node.rb

```diff
@@ -29,7 +29,11 @@ def class_or_module_name(class_or_module_node)
           # (module (const nil :Foo) (nil))
           #   "module Foo; end"
           identifier = class_or_module_node.children[0]
-          constant_name(identifier)
+          begin
+            constant_name(identifier)
+          rescue Node::TypeError
+            nil
+          end
```

Comment:
I don't think we want to rescue against a TypeError in this method, because we expect it to raise a TypeError as seen on line 38. I would suggest rescuing the type of error in the method that calls this method.

Reply:
Makes sense!